hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightEndpoint.java

```diff
@@ -211,10 +223,45 @@ public Response getOpenKeyInfo(
         break;
       }
     }
+    // Populate the clusterSummary map
+    createClusterSummary(clusterSummary);
+
+    openKeyInsightInfo.setClusterSummary(clusterSummary);
+
     openKeyInsightInfo.setLastKey(lastKey);
     return Response.ok(openKeyInsightInfo).build();
   }
 
+  private void createClusterSummary(Map<String, Object> clusterSummary) {
+    try {
+      long replicatedSizeOpenKey = globalStatsDao.findById(OmTableInsightTask
+          .getReplicatedSizeKeyFromTable(OPEN_KEY_TABLE)).getValue();
+      long replicatedSizeOpenFile = globalStatsDao.findById(OmTableInsightTask
+          .getReplicatedSizeKeyFromTable(OPEN_FILE_TABLE)).getValue();
+      long unreplicatedSizeOpenKey = globalStatsDao.findById(OmTableInsightTask
+          .getUnReplicatedSizeKeyFromTable(OPEN_KEY_TABLE)).getValue();
+      long unreplicatedSizeOpenFile = globalStatsDao.findById(OmTableInsightTask
+          .getUnReplicatedSizeKeyFromTable(OPEN_FILE_TABLE)).getValue();
+      long openKeyCountForKeyTable = globalStatsDao.findById(OmTableInsightTask
+          .getTableCountKeyFromTable(OPEN_KEY_TABLE)).getValue();
+      long openKeyCountForFileTable = globalStatsDao.findById(OmTableInsightTask
+          .getTableCountKeyFromTable(OPEN_FILE_TABLE)).getValue();
+
+      // Calculate the total number of open keys
+      clusterSummary.put("totalOpenKeys",
+          openKeyCountForKeyTable + openKeyCountForFileTable);
+      // Calculate the total replicated and unreplicated sizes
+      clusterSummary.put("totalReplicatedDataSize",
```

Comment:
Based on these naming of attributes, please change the "`replicatedTotal`" and "`unreplicatedTotal`" attributes names as well to keep consistency in "`KeyInsightInfoResponse`" class. We can change to "`replicatedDataSize`" and "`unreplicatedDataSize`"

Reply:
Done changed the attributes `replicatedTotal` & `unreplicatedTotal` to `replicatedDataSize` & `unreplicatedDataSize`